pushover/README.md

```diff
@@ -0,0 +1,85 @@
+# Pushover  
+
+Pushover is one out of many push services, which is compatible with Android, IOS and Windows Clients. This plugin make it possible to push messages to your clients from smarthomeNG.
+  
+## Requirements
+  
+### Python libraries  
+* requests - [install instructions](http://docs.python-requests.org/en/latest/user/install/#install "http://docs.python-requests.org/en/latest/user/install/#install")
```

Comment:
I think someone wanting to use a plugin should find the requirements (hard- and software) in one place (the README.md). The requirements section doesn't need to specify the exact version(s). It can refer to the requirements file.

I think a lot of users don't now Python very well (of not at all) and have no clue about the requirements file. If the requirement of a module is documented in the README with a reference to the requirements file (regarding to the version), all user would be helped.